modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -24,25 +23,40 @@ and the `bmc` parameter for the `install-config.yaml` file.
 | `UEFI`
 | The boot mode for a node. Options are `legacy`, `UEFI`, and `UEFISecureBoot`. If `bootMode` is not set, Ironic sets it while inspecting the node.
 
-| `bootstrapExternalStaticDNS`
+a| 
+----
+platform:
+  baremetal: 
+    `bootstrapExternalStaticDNS`
```

Comment:
I think the backticks are leftover from when this was a line line and that they don't actually belong in the YAML, right?
```suggestion
    bootstrapExternalStaticDNS
```